packages/story-editor/src/app/api/apiProvider.tsx

```diff
@@ -17,19 +17,22 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { useCallback, useRef } from '@googleforcreators/react';
-import { getAllTemplates } from '@googleforcreators/templates';
+import type { ReactNode } from 'react';
+import { getAllTemplates, Template } from '@googleforcreators/templates';
 
 /**
  * Internal dependencies
  */
 import { useConfig } from '../config';
 import Context from './context';
 
-function APIProvider({ children }) {
+interface APIProviderProps {
+  children: ReactNode;
+}
+function APIProvider({ children }: APIProviderProps) {
   const { apiCallbacks: actions, cdnURL } = useConfig();
-  const pageTemplates = useRef({
+  const pageTemplates = useRef<{ base: Template[] }>({
```

Comment:
Why is this ref not just a `Template[]`?

Also, if it was, you'd need perhaps need to type it as `useRef<Template[] | null>(...)`.